grpc/binlog/v1alpha/binarylog.proto

```diff
@@ -64,6 +68,26 @@ message GrpcLogEntry {
 
   // true if payload does not represent the full message or metadata.
   bool truncated = 7;
+
+  // The method name. Logged for the first entry:
+  // RECV_INITIAL_METADATA for server side or
+  // SEND_INITIAL_METADATA for client side.
+  string method_name = 8;
+
+  // status_code and status_message:
+  // Only present for SEND_TRAILING_METADATA on server side or
+  // RECV_TRAILING_METADATA on client side.
+  uint32 status_code = 9;
+  // An original status message before any transport specific
+  // encoding.
+  string status_message = 10;
+
+  // the timeout in nanoseconds
+  google.protobuf.Duration timeout_ns = 11;
+
+  // The entry sequence id for this call. The first GrpcLogEntry has
+  // a value of 1, to disambiguate from an unset value.
+  uint32 sequence_id_within_call = 12;
```

Comment:
The comment for this should try to make it clear that the purpose for this is to allow the log reader to determine when there are missing entries.